corehq/apps/app_manager/templates/app_manager/v2/form_view_base.html

```diff
@@ -211,7 +211,7 @@
 
         gaTrackLink($('.breadcrumb .view-form'), 'App Builder', 'View Form', 'Breadcrumb');
 
-        {% if allow_cloudcare %}
+        {% if can_preview_form and allow_cloudcare %}
```

Comment:
This isn't the same place as was edited in the V1 file. I think you should add this edit to the V1 file and then also hide this section in the V2 file: https://github.com/dimagi/commcare-hq/blob/a5c29f2cacafcc9bd2651223d4067fa537d6c7eb/corehq/apps/app_manager/templates/app_manager/v2/form_view_base.html#L276-L281

There are also other places where you can launch the App Preview from. Search for "Preview App" and "preview_app.js".

@biyeun @orangejenny FYI that this will also affect app manager V2. Since V2 isn't released yet let us know if you'd rather handle this in a different way.